staging/src/kubevirt.io/client-go/kubecli/vmi.go

```diff
@@ -499,5 +499,6 @@ func (v *vmis) VSOCK(name string, options *v1.VSOCKOptions) (StreamInterface, er
 	}
 	queryParams := url.Values{}
 	queryParams.Add("port", strconv.FormatUint(uint64(options.TargetPort), 10))
+	queryParams.Add("tls", strconv.FormatBool(options.UseTLS))
```

Comment:
Can we default to true?

Reply:
Done. It is now the default on the API and in the client. People have to disable it now explicitly. Then they can also add custom e2e encryption.